qa/tasks/cephfs/test_fragment.py

```diff
@@ -314,3 +314,41 @@ def _count_fragmented():
             lambda: _count_fragmented() > 0,
             timeout=30
         )
+
+    def _run_dir_frag(self, killpointv):
+
+        status = self.fs.status()
+
+        rank_0 = self.fs.get_rank(status=status, rank = 0)
+        rank_0_name = rank_0['name']
+
+        self.fs.set_config("mds_kill_dirfrag_at", str(killpointv), rank=0, status=status)
+
+        try:
+            self.test_oversize()
+        except Exception as e:
+            log.error(e.__str__())
```

Comment:
If `test_oversize` fails, wouldn't we want that to also fail this killpoint test?